cmd/config/docs/commands/merge3.md

```diff
@@ -0,0 +1,25 @@
+## merge3
+
+[Alpha] Merge Resource configuration files (3-way)
+
+### Synopsis
+
+[Alpha] Merge Resource configuration files (3-way)
+
+Merge (3-way) reads Kubernetes Resource yaml configuration files from source packages and updated 
```

Comment:
Merge3 performs a 3-way merge by applying the diff between 2 sets of Resources to a 3rd set.

Merge3 may be for rebasing changes to a forked set of configuration -- e.g. compute the difference between the original set of Resources that was forked and an updated set of those Resources, then apply that difference to the fork.